test/db.js

```diff
@@ -217,34 +217,30 @@ test('Berlin Jungfernheide to ATZE Musiktheater', co(function* (t) {
 	t.end()
 }))
 
-test('journeys: via works – changing trains', co(function* (t) {
-	// Going from Westhafen to Wedding via Württembergalle without changing
-	// is currently impossible. We check if the routing engine lets us change
-	// at Württembergallee if we set it as via.
+test('journeys: via works – with detour', co(function* (t) {
+	// Going from Westhafen to Wedding via Württembergalle without detour
+	// is currently impossible
```

Comment:
I think we should re-add "We check if the routing engine compute a detour." here.

Reply:
👍